Resources/doc/cache-resolver/web_path.rst

```diff
@@ -5,7 +5,21 @@ Web Path Resolver
 =================
 
 The ``WebPathResolver`` resolver enabled cache resolution using the
-web path of your application.
+web path of your application. The resulting web path is computed by
+taking a number of factors into account, including the `request context`_
+as provided by the Symfony HTTP kernel.
+
+.. tip::
+
+    The request context is most notably used to determine the HTTP scheme
+    prepended to the final URL. If you use a proxy to offload TLS traffic
+    decryption and need the resolver to generate HTTPS URLS, you will need
```

Comment:
Heh; already noticed that myself in my follow-up commit. though do please take a look at my final changes and advise if you notice anything else needing cleanup.